pkg/cmd/roachtest/test_runner.go

```diff
@@ -926,6 +933,34 @@ func (r *testRunner) runWorker(
 	}
 }
 
+// destroyCluster runs cluster destroy in a goroutine and adds 1 to the wait group if the wsit is false
+// if the wait is true, it waits till the cluster is destroyed
+// if the cluster is nil, it just returns
+func (r *testRunner) destroyCluster(
+	ctx context.Context,
+	clusterDestroyWg *sync.WaitGroup,
+	c *clusterImpl,
+	l *logger.Logger,
+	wait bool,
```

Comment:
Instead of passing a bool to cover this edge case, what if you just called `c.Destroy(context.Background(), closeLogger, l)` directly when you don't want to wait?